impl/sql/sequenced/sequenced.go

```diff
@@ -91,9 +95,9 @@ func create(db *sql.DB) error {
 	return nil
 }
 
-func (s *Sequenced) insertMapRow(txn transaction.Txn, mapID int64) error {
+func insertMapRow(db *sql.DB, mapID int64) error {
```

Comment:
You can make this a member of Sequenced object instead of passing db and mapID. This makes it consistent with the rest of the code. For example:
https://github.com/google/keytransparency/blob/master/impl/sql/mutations/mutations.go#L122